src/skmatter/sample_selection/_base.py

```diff
@@ -342,7 +342,6 @@ class CUR(_CUR):
     ...         [-0.03, -0.53, 0.08],
     ...     ]
     ... )
-    >>> np.random.seed(0)  # there is a source of randomness in it
     >>> selector.fit(X)
     CUR(n_to_select=2)
     >>> np.round(selector.pi_, 2)  # importance scole
```

Comment:
On my machine I still get

```
Expected:
    array([0., 1., 0.])
Got:
    array([0.23, 0.  , 0.77])
```

for the output.

Reply:
thanks for checking, I don't have a better idea than going back to debugging together comparing output of sparse svd on your and my machine

Reply:
Okay then lets do it together next week.